### PR TITLE
Webview2

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Related to https://github.com/OfficeDev/office-js/issues/1399

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1fc68d1](https://github.com/OfficeDev/office-js-docs-pr/commits/1fc68d1a14553474a4726740a94adabb1fe55128)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview2/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins?branch=pr-en-us-2215) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009222356359877-2215%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=piNrpvCo%2BqnxwykzjbkdfSjs4vP2KluCK2gB5E7xSUg%3D&st=2020-09-22T23%3A52%3A10Z&se=2020-10-23T23%3A57%3A10Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by alison-mk — APPROVED** [review]
Looks good to me!

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Which browser is used depends on:
 
 The following table shows which browser is used for the various platforms and operating systems.
 
-|OS|Office version|Edge WebView2 (Chromium-based) installed?|Browser|
+|OS|Office version|Edge WebView2 (Chromium-based) installed?<sup>5</sup>|Browser|
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/browsers-used-by-office-web-add-ins.md:19]: @Rick-Kirkham Recommend renumbering in order of appearance.

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 2]
```diff
@@ -28,15 +28,18 @@ The following table shows which browser is used for the various platforms and op
...
+<sup>4</sup> The browser that is used for this combination of versions depends on the update channel of the Microsoft 365 subscription. If the user is on the [Beta channel](https://insider.office.com/join/windows) (formerly Insiders Fast channel), then Office uses Microsoft Edge with WebView2 (Chromium-based). For any other channel, Office uses Microsoft Edge with the original WebView (EdgeHTML). Support for WebView2 in other channels is expected in early 2021. *See also note 5*.
+
+<sup>5</sup> The embeddable WebView2 control must be installed in addition to the installation of Microsoft Edge so that Office can embed it. To install it, see [Microsoft Edge WebView2 (Preview) / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/en-us/microsoft-edge/webview2/).
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/browsers-used-by-office-web-add-ins.md:41]: Should exclude en-us

**Pull request merged**